lldb/source/Host/posix/ProcessLauncherPosixFork.cpp

```diff
@@ -193,8 +192,13 @@ struct ForkLaunchInfo {
     }
 
     // Start tracing this child that is about to exec.
+#if !defined(_AIX)
```

Comment:
Generally I would prefer positive conditions as the first part of an if. So here it would be if AIX use ptrace64, else use ptrace.

Also I would keep the body of the if common:
```
#ifdef AIX
  if (ptrace64(...))
#else
  if (ptrace(...))
#endif
    ExitWithError(...);
```